src/commands/commands.js

```diff
@@ -3,15 +3,22 @@
  * See LICENSE in the project root for license information.
  */
 
-Office.onReady(info => {
-  // If needed, Office.js is ready to be called
-});
+var config;
+var configEvent;
+
+// The initialize function must be run each time a new page is loaded.
+Office.initialize = function (reason) {
+  config = getConfiguration();
+};
 
 function showConfigDialog(event) {
   // Not Configured: Show the configuration dialog
   configEvent = event;
   var url = new URI('../settings/login.html').absoluteTo(window.location).toString();
   var dialogOptions = { width: 40, height: 60, displayInIframe: true };
+  if (config) {
+    url = url + '?param=' + JSON.stringify(config);
```

Comment:
Around `JSON.stringify()` there should be `encodeURIComponent()` as well to prevent some characters to be misinterpreted.